modules/ballerina-test/src/test/resources/test-src/statements/services/nativeimpl/inbound/response/in-response-compile-negative.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina.net.http;
+
+function testResponseSetStatusCode (http:InResponse res, string statusCode) (http:InResponse) {
+    res.statusCode = statusCode;
+    return res;
+}
+
+function testResponseGetMethod (http:InResponse res) (string) {
+    string method = res.getMethod();
```

Comment:
shall we remove this form the response ? because we can get it directly from the response struct.

Reply:
Removed getMethod(), getRequestURL and GetStatusCode